src/System.IO.Pipes/src/System/IO/Pipes/PipeStream.cs

```diff
@@ -477,7 +477,7 @@ internal virtual void CheckPipePropertyOperations()
         // read returns 0 bytes
         [SecurityCritical]
         [SuppressMessage("Microsoft.Security", "CA2122:DoNotIndirectlyExposeMethodsWithLinkDemands", Justification = "Consistent with security model")]
```

Comment:
These SuppressMessage attributes can be deleted along with the security attributes.